e2e-tests/insurance/cypress/e2e/journeys/your-buyer/outstanding-or-overdue-payments/outstanding-or-overdue-payments.spec.js

```diff
@@ -67,22 +68,38 @@ context(
 
       describe(TOTAL_OUTSTANDING_PAYMENTS, () => {
         it(`should render a label for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-          cy.checkText(field(TOTAL_OUTSTANDING_PAYMENTS).label(), FIELDS[TOTAL_OUTSTANDING_PAYMENTS].LABEL);
+          cy.assertCopyWithCurrencyName({
+            expectedCopy: FIELDS[TOTAL_OUTSTANDING_PAYMENTS].LABEL,
+            currencyName: GBP.name,
+            selector: field(TOTAL_OUTSTANDING_PAYMENTS).label(),
+          });
         });
 
         it(`should render an input for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
           field(TOTAL_OUTSTANDING_PAYMENTS).input().should('be.visible');
         });
+
+        it(`should render a prefix for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
+          cy.checkText(field(TOTAL_OUTSTANDING_PAYMENTS).prefix(), SYMBOLS.GBP);
```

Comment:
dereference SYMBOLS.GBP as used twice?

Reply:
I don't think any `SYMBOLS` should be dereferenced because having `GBP` implies it is not a symbol. We have other things like currency names and codes called `GBP`. `SYMBOLS.GBP` makes it abundantly clear IMO.